tests/testsuite/build.rs

```diff
@@ -268,7 +268,7 @@ fn cargo_compile_with_invalid_version() {
 [ERROR] failed to parse manifest at `[..]`
 
 Caused by:
-  Expected dot for key `package.version`
+  data did not match any variant of untagged enum MaybeWorkspace for key `package.version`
```

Comment:
[I realize now this is a regression.](https://github.com/rust-lang/cargo/commit/cb53e1b2056a02bd1eb83d926e1cc37e56180368) Will need to find a better way of handling this.